libtransmission/torrent.h

```diff
@@ -669,6 +669,21 @@ struct tr_torrent final : public tr_completion::torrent_view
         torrent's content than any other mime-type. */
     [[nodiscard]] std::string_view primaryMimeType() const;
 
+    constexpr void setSequentialDownload() noexcept
+    {
+        this->sequentialDownload = true;
+    }
+
+    constexpr void setRandomDownload() noexcept
+    {
+        this->sequentialDownload = false;
+    }
```

Comment:
I kind of don't like the `random` terminology since (a) pedantically speaking we _do_ give weight to nearly-complete pieces, rarer pieces, etc. so it's not fully random, and (b) it doesn't add any value to the API. IMO let's replace these two functions with a single setter that takes a bool arg

```suggestion
    constexpr void setSequentialDownload(bool is_sequential) noexcept
    {
        this->sequentialDownload = is_sequential;
    }
```

...and then remove `TR_KEY_randomDownlownload`, update the RPC, etc. to reflect this change

Reply:
agreed - I copied the semantic from the patch. thanks for the patch !